src/drawing/__tests__/DrawingPathGroup-test.tsx

```diff
@@ -55,5 +55,18 @@ describe('DrawingPathGroup', () => {
             expect(wrapper.children()).toHaveLength(0);
             expect(wrapper.exists('path')).toBe(false);
         });
+
+        test('should use render function if provided', () => {
+            const {
+                stroke: { size },
+            } = pathGroups[0];
+            const expectedBorderWidth = size + DRAWING_BORDER_WIDTH * 2;
```

Comment:
This is essentially duplicating the logic from the component. Should we hardcode the output we expect given a known input, instead ?